ui/pages/reservation-unit/[id].tsx

```diff
@@ -1049,140 +1043,105 @@ const ReservationUnit = ({
+                                  n.value ===
+                                  reservationUnit.reservationsMaxDaysBefore
+                              )?.label,
+                              unit: t(
+                                `reservationUnit:reservationInfo1-${
+                                  reservationUnit.reservationsMaxDaysBefore ===
+                                  14
+                                    ? "weeks"
+                                    : "months"
+                                }`
+                              ),
+                            }}
+                            components={{ bold: <strong /> }}
+                          />
                         )}
                       {reservationUnit.reservationsMaxDaysBefore === 0 &&
                         reservationUnit.reservationsMinDaysBefore > 0 && (
-                          <Trans i18nKey="reservationUnit:reservationInfo1-3">
-                            Voit tehdä varauksen{" "}
-                            <strong>
-                              viimeistään{" "}
-                              {{
-                                reservationsMinDaysBefore:
-                                  reservationUnit.reservationsMinDaysBefore,
-                              }}{" "}
-                              päivää etukäteen
-                            </strong>
-                            .
-                          </Trans>
+                          <Trans
+                            defaults="Voit tehdä varauksen <bold>viimeistään {{reservationsMinDaysBefore}} päivää etukäteen</bold>."
+                            values={{
+                              reservationsMinDaysBefore:
+                                reservationUnit.reservationsMinDaysBefore,
+                            }}
+                            components={{ bold: <strong /> }}
+                          />
                         )}
                     </p>
                   )}
                   {reservationUnit.reservationEnds && (
                     <p>
-                      <Trans i18nKey="reservationUnit:reservationInfo2">
-                        <strong>
-                          Varauskalenteri on auki{" "}
-                          {{
-                            reservationEnds: formatDate(
-                              reservationUnit.reservationEnds
-                            ),
-                          }}
-                        </strong>{" "}
-                        asti.
-                      </Trans>
+                      <Trans
+                        defaults="Varauskalenteri on auki <bold>{{reservationEnds}}</bold> asti."
+                        values={{
+                          reservationEnds: formatDate(
+                            reservationUnit.reservationEnds
+                          ),
+                        }}
+                        components={{ bold: <strong /> }}
+                      />
```

Comment:
Missing i18nKey. Need to refactor the translation tags.
See https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0